package.json

```diff
@@ -40,28 +40,25 @@
     },
     "dependencies": {
         "@azure/functions": "^1.2.3",
-        "@types/lodash": "^4.14.119",

```

Comment:
No references found in *.d.ts file - moved to a dev dependency
<img width="591" alt="Screen Shot 2022-04-22 at 2 44 17 PM" src="https://user-images.githubusercontent.com/11282622/164802645-7be3194b-a3d8-49b7-96d2-581b20063b07.png">